### ISSUE TITLE
Auto focusing map on posts is unexpected when theres just 1 or 2 posts

### ISSUE THREAD

**Issue by rjmackay** [ushahidi/platform#1736]
### Expected behaviour
Users first setting up a deployment expect the default map settings to control their map view.

### Actual behaviour
Default map settings are only used when there are *no* posts. As soon as the users adds 1 post the map focuses on that. This is weird since it zooms right in.

**Comment by rjmackay** [comment]
https://app.intercom.io/a/apps/hl5rfiga/respond/inbox/unassigned/conversations/9649146068

**Comment by ryanchristo** [comment]
@rjmackay I submitted pull request ushahidi/platform-client#679, which removes the auto-focus. I started thinking afterwards that this might be a good optional feature that the user could toggle on and off in general settings rather than choosing the default location and zoom. Thoughts?

**Comment by rjmackay** [comment]
@ryanchristo Agreed. I don't think we want to remove auto focus entirely... its still a sensible behaviour in a lot of cases, but the current combination of configuring a default location is confusing.

@jshorland @justinscherer any thoughts on the best resolution to this issue?

Theres an older issue thats relevant here: #1339 - this basically specc'd:
- Remove the default location setting entirely
- Add a per-field settings for the default location used when inputting data

If you still thats the best course of action, maybe @ryanchristo could help build that out instead.

**Comment by ryanchristo** [comment]
@rjmackay I like the solution in #1339 and would be happy to help build it out.

The map would still zoom in after the first post if the custom location field settings went unchanged ("Use this location to locate responses" being on by default) but in this case it would be much more clear why this is happening. This would prevent the confusion caused by the current default location option in General Settings (which would be removed) currently being overridden by the auto-focus and appearing to do nothing.

**Comment by ryanchristo** [comment]
As mentioned in https://github.com/ushahidi/platform/issues/1339#issuecomment-314987607. We can set a max zoom when there are less than 10 posts. This would be a quick fix once we decide on a value.

**Comment by jrtricafort** [comment]
@justinscherer would love to have a quick conversation with you to get up to speed on this and https://github.com/ushahidi/platform/issues/1339. I think it may be worth bringing back from the Icebox and prioritizing as a quick fix in one of the next couple cycles. I'll connect with you on Slack next week.

**Comment by jrtricafort** [comment]
Scroll down to the last ~5 comments on this issue to see a spec for a potential fix: https://github.com/ushahidi/platform/issues/1339

TL;DR: Remove Default Map from General Settings, or make it more explicit to user how and when this Default Map gets activated vs. when AutoZoom is activated.

**Comment by jrtricafort** [comment]
Duplicate of #1339

**Comment by rowasc** [comment]
Closing in favor of #1339
(per Juan's comment)
